client/lib/signup/step-actions.js

```diff
@@ -167,7 +165,7 @@ export function createSiteWithCart(
 				title: siteTitle,
 			},
 		},
-		public: -1,
+		public: getNewSitePublicSetting( { ...dependencies, ...stepData } ),
```

Comment:
It looks like this call will always return `-1`, because it's passing an object into just the first parameter of `getNewSitePublicSetting`, so in that function, `stepData` will be undefined.

Reply:
Good observation from @andrewserong:

```suggestion
		public: getNewSitePublicSetting( { ...dependencies }, { ...stepData } ),
```

Is there a reason for the clone here again?

```suggestion
		public: getNewSitePublicSetting( dependencies, stepData ),
```